saloni.md

```diff
@@ -0,0 +1,17 @@
+WorkLog of Saloni
+==============
+
+###29 Feb, 2016
+This contains the work done till 29 feb, 2016.
+
+####Work
```

Comment:
@salonisj18 
SAMPLE.md has single \# for date, I would suggest if you could follow that.